packages/core/src/components/EditableTypography/EditableTypography.module.scss

```diff
@@ -1,11 +1,12 @@
 @import "~monday-ui-style/dist/mixins";
 
 .editableTypography {
-  display: inline-flex;
+  display: block;
```

Comment:
**note:** 

I still changed display to block because it was requested.

Reply:
Seems like this causes some issues, see video attached:

https://github.com/user-attachments/assets/733a6231-3798-469c-afe8-b1c4a0dd0f05

Reply:
@talkor 

Thank you for testing! I think changing display to 'block' caused that behavior. I reverted this change and I this issue is fixed for me locally  - I verified it locally and ran lint & tests. Does proposed change fix the issue for you?